coderd/database/migrations/000098_template_version_variables.up.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS template_version_variables (
+    template_version_id uuid not null references template_versions (id) on delete cascade,
+    name text not null,
+    description text not null,
+    type text not null,
```

Comment:
I saw this was listed as an enum in the API, should we turn it into an enum in the DB as well?